### PR TITLE
ci(perf-measures): support `KB`

### PR DESCRIPTION
**Pull request by EdamAme-x**
We can check `kb`

### CODE REVIEW

**Comment by EdamAme-x** [comment]
hmm?
https://github.com/honojs/hono/actions/runs/11945102648/job/33297187610?pr=3696#step:7:95
@k2tzumi What do you know?

**Comment by EdamAme-x** [comment]
> * [JS Bundle Analysis](https://docs.codecov.com/docs/javascript-bundle-analysis) - Avoid shipping oversized bundles

Interesting 👀

File: perf-measures/bundle-check/scripts/check-bundle-size.ts [hunk 1]
```diff
@@ -19,12 +19,21 @@ async function main() {
...
+      key: 'bundle-size-b',
+      name: 'Bundle Size (B)',
       value: bundleSize,
       unit: 'B',
     })
+
+    metrics.push({
+      key: 'bundle-size-kb',
+      name: 'Bundle Size (KB)',
+      value: (bundleSize / 1024).toFixed(2),
```
Comment (k2tzumi) [inline perf-measures/bundle-check/scripts/check-bundle-size.ts:33]: I don't know, as I have never tried it, but I was curious about the following warning message.

https://github.com/honojs/hono/actions/runs/11945102648/job/33297187610?pr=3696#step:7:94
`Skip collecting custom metrics: json: float unexpected end of JSON input`

What if we set the value as a string?

Comment (EdamAme-x) [reply perf-measures/bundle-check/scripts/check-bundle-size.ts]: This is the only issue that has the same error, not octocov.  
So this error itself is unusual...  
Hmmm...  

https://github.com/pluja/whishper/issues/78

File: perf-measures/bundle-check/scripts/check-bundle-size.ts [hunk 2]
```diff
@@ -19,12 +19,21 @@ async function main() {
...
+      key: 'bundle-size-b',
+      name: 'Bundle Size (B)',
       value: bundleSize,
       unit: 'B',
     })
+
+    metrics.push({
+      key: 'bundle-size-kb',
+      name: 'Bundle Size (KB)',
+      value: (bundleSize / 1024).toFixed(2).toString(),
```
Comment (k2tzumi) [inline perf-measures/bundle-check/scripts/check-bundle-size.ts:33]:
```suggestion
      value: parseFloat((bundleSize / 1024).toFixed(2)),
```
It looks like it had to be Number in the custom metrics schema.
https://github.com/k1LoW/octocov/blob/main/report/custom_metrics_schema.json#L51-L53

**Comment by EdamAme-x** [comment]
It seems to work well

**Comment by yusukebe** [comment]
Hi @EdamAme-x

Good PR! One thing I noticed.

When displaying Bundle Size, I think it would be easier to compare "K" with "B":

CleanShot 2024-11-22 at 09 21 29@2x

In the above, the "B" and "KB" digits appear to be aligned. `ls -lh` command also uses the method I suggest, so it's okay.

CleanShot 2024-11-22 at 09 25 58@2x

**Comment by EdamAme-x** [comment]
thanks for suggestions

**Review by yusukebe — COMMENTED** [review]
LGTM!

**Comment by yusukebe** [comment]
@EdamAme-x 

Thank! Merging.

**Pull request merged**